xml/gl.xml

```diff
@@ -9180,7 +9180,7 @@ typedef unsigned int GLhandleARB;
             <param group="CheckedInt32"><ptype>GLint</ptype> <name>level</name></param>
             <param group="CheckedInt32"><ptype>GLint</ptype> <name>xoffset</name></param>
             <param><ptype>GLsizei</ptype> <name>width</name></param>
-            <param group="PixelFormat"><ptype>GLenum</ptype> <name>format</name></param>
+            <param group="InternalFormat"><ptype>GLenum</ptype> <name>format</name></param>
```

Comment:
It's awesome to see that vendors are starting to pitch in on the groups!

This makes sense, but I think we need to establish a breaking change policy for groups going forwards. First Google and now Arm, it looks like even though Khronos themselves don't use the groups they're becoming of increasing importance among the members therein.

Other than that fine with this.

Reply:
If you're able to provide details on your use case for the groups this'd help, but no worries if not. I'll open a separate issue for establishing this breaking change policy.

Reply:
It's for pretty-printing traced API calls so that GLenum/GLbitfields appear with their _name_ instead of their _integer_ value. gl.xml is used for generating a lookup table that makes this conversion possible.

GLenum value to name is not a one-to-one mapping (e.g. `0` can be `GL_POINTS` or `GL_ZERO`). The parameter groups make it possible to retrieve the relevant set of GLenum names for a specific parameter of a specific function.